src/internal/m365/collection/drive/helper_test.go

```diff
@@ -384,6 +395,14 @@ func (ecs expectedCollections) compareColl(t *testing.T, coll data.BackupCollect
 		p       = fullOrPrevPath(t, coll)
 	)
 
+	// check the metadata collection separately
+	if coll.FullPath() != nil && coll.FullPath().Equal(defaultMetadataPath(t)) {
```

Comment:
nit: could this be simplified to just `p` since that's going to be either the full path or the previous path?
```suggestion
	if p.Equal(defaultMetadataPath(t)) {
```

Reply:
seems reasonable